src/turfs/katmos.rs

```diff
@@ -785,11 +785,13 @@ pub(crate) fn equalize(
 				}) {
 				return None;
 			}
-			let maybe_m = turf_gases().get(&i);
-			if maybe_m.is_none() {
-				return None;
-			}
-			let m = *maybe_m.unwrap();
+			let m = {
```

Comment:
```suggestion
			turf_gases().try_get(&i).try_unwrap().and_then(|m| {
```
basically, this is what you should be doing instead of the nested `return None;` thing; put the flood_fill_equalize_turfs into the closure here

Reply:
I don’t want to hold a ref into the map here

Reply:
considering it will be holding it for the entire closure including floodfill